guides/WixApps.md

```diff
@@ -1,20 +1,26 @@
-# About Wix Business Solutions
+# About Apps Made By Wix
+
+Wix sites can contain one or more apps made by Wix. Some API calls require a reference to the specific app made by Wix that it relates to. For example, when calling the [Create Comment](https://dev.wix.com/docs/rest/api-reference/comments/comments/create-comment) API, you must pass an `appId` to specify where to create the comment, for example, in Wix Blogs. Additionally, some API calls return a reference to the app made by Wix. For example, calling the [`Get a Coupon`](https://dev.wix.com/docs/rest/api-reference/coupons/coupons/get-a-coupon) API returns a coupon object with an `appId` referring to the ID of the Wix app that created the coupon, such as Wix Bookings or Wix Events. 
+
+See the following apps made by Wix and their app IDs:
+
+| **App made by Wix**    | **app ID**                           |
+|------------------------|--------------------------------------|
+| Wix eCommerce          | 1380b703-ce81-ff05-f115-39571d94dfcd |
+| Wix Stores             | 215238eb-22a5-4c36-9e7b-e7c08025e04e |
+| Wix Bookings           | 13d21c63-b5ec-5912-8397-c3a5ddb27a97 |
+| Wix Events             | 140603ad-af8d-84a5-2c80-a0f60cb47351 |
+| Wix Paid Plans         | 1522827f-c56c-a5c9-2ac9-00f9e6ae12d3 |
+| Wix Pay Button         | 3575d251-42c3-4992-adff-170b2af90a2c |
```

Comment:
What is Wix Pay Button? I never heard of this one.  
Perhaps this is Wix Payments?  Either Adam or Aliza might know.